### DISCUSSION TITLE
add a11y utilities on dev

### DISCUSSION THREAD

**Discussion by userquin** [unocss/unocss#549] [Ideas]
I'm collecting some dev utilities to enhance a11y on development (I will add here more when I've time ;) ), maybe adding some css file via virtual module to control dev environment (`import 'uno-a11y.css'`):
- image without alt: apply rule similar to `question-mark` with 5px red bouncing border
- input without label: maybe complex case
- form without submit button or input type submit (we'll need to rely on `:has`)

I need to review a few tools, so be patient, add your ones, we can also send a tweet for feedback.

@antfu @chu121su12

**Comment by chu121su12** [comment]
Some more rules that I think should get `question-mark`:
- button without explicit type (becomes submit inside form)
- input without explicit type
- head without title/empty title
- a without href/empty href

**Comment by JessicaSachs** [comment]
Maybe we can use the lists that [axe-core](https://github.com/dequelabs/axe-core) keeps in order to get good ideas and then build UI to support the failure states. [Rule Descriptions](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md).

**Comment by userquin** [comment]
accesslint.com
